Velaptor/Batching/IBatchingManager.cs

```diff
@@ -16,26 +17,30 @@ internal interface IBatchingManager
     /// </summary>
     /// <param name="item">The item to add.</param>
     /// <param name="layer">The layer to add the item on.</param>
```

Comment:
Refactor to remove "on" at the end. 
```suggestion
    /// <param name="layer">The layer to add the item.</param>
```